api/v1alpha1/healthcheck_types.go

```diff
@@ -104,6 +106,12 @@ type URLArtifact struct {
 	VerifyCert bool   `json:"verifyCert,omitempty"`
 }
 
+// SchedulerSpec contains the cron expression
+type SchedulerSpec struct {
```

Comment:
related to prev comment
```suggestion
type ScheduleSpec struct {
```